assets/css/frontend-form/layout2.css

```
@@ -216,100 +246,361 @@ body.mceContentBody{
+/* ==== Figma deep-sync (node 5527:36594) — exact navy palette & geometry ====
+   Every value below is taken from the dark-navy template frame itself; no
+   tokens borrowed from other templates. */
+.wpuf-form-add.wpuf-form-layout2 {
+  --wpuf-l2-container-bg: #1B2431;
+  --wpuf-l2-surface-bg: #222C3C;
+  --wpuf-l2-border: #344156;
+  --wpuf-l2-label: #9CABC3;
+  --wpuf-l2-primary: #3B82F6;
+  --wpuf-l2-help: #4A5971;
+  --wpuf-l2-accent: #3B82F6;
+  padding: 40px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)